{{cookiecutter.project_name}}/.github/workflows/build-test-package.yml

```diff
@@ -4,9 +4,9 @@ on: [push,pull_request]
 
 jobs:
   cxx-build-workflow:
-    uses: InsightSoftwareConsortium/ITKRemoteModuleBuildTestPackageAction/.github/workflows/build-test-cxx.yml@df1df6398389fa1a99d169b1264b9647d2164dd0
+    uses: InsightSoftwareConsortium/ITKRemoteModuleBuildTestPackageAction/.github/workflows/build-test-cxx.yml@v5.4.0
 
   python-build-workflow:
-    uses: InsightSoftwareConsortium/ITKRemoteModuleBuildTestPackageAction/.github/workflows/build-test-package-python.yml@df1df6398389fa1a99d169b1264b9647d2164dd0
+    uses: InsightSoftwareConsortium/ITKRemoteModuleBuildTestPackageAction/.github/workflows/build-test-package-python.yml@v5.4.0
```

Comment:
Same here.

Reply:
@SimonRit should we just merge those into the `v5.4.0` branch?

Reply:
> @SimonRit should we just merge those into the `v5.4.0` branch?

Sure, I hadn't realized that `v5.4.0` is a branch and not a tag :-).

Reply:
Looks like a [tag](https://github.com/InsightSoftwareConsortium/ITKPythonPackage/releases/tag/v5.4.0) to me, would you move it then?

Reply:
@SimonRit corresponding PR here: https://github.com/InsightSoftwareConsortium/ITKRemoteModuleBuildTestPackageAction/pull/85